storage.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	model "github.com/golang-bristol/beer-model"
+)
+
+const (
+	// StorageTypeJSON is json type indicator
+	StorageTypeJSON = "JSON" // storage_json.go
```

Comment:
For this scenario, I think this is a good idea. I like it. However if/when added to a configuration file this might not be easily readable. We can think about this later when the time comes.

Reply:
Or we can worry about the configuration file when the time comes ;)